bl/evaluation/evaluator.go

```diff
@@ -34,10 +34,11 @@ type EvaluationResults struct {
 	}
 }
 
-func (e *Evaluator) CreateEvaluation(cliId string, cliVersion string, k8sVersion string) (*cliClient.CreateEvaluationResponse, error) {
+func (e *Evaluator) CreateEvaluation(cliId string, cliVersion string, k8sVersion string, policyName string) (*cliClient.CreateEvaluationResponse, error) {
```

Comment:
I wonder if it's time to group some of the parameters, maybe something like `cliInfo, options` this will be much handy to send all the flags as options

Reply:
I agree, but this can turn out into a heavy change so I think I'll leave it to a smaller pr :)